src/ui/settings/ViewLanguage.qml

```diff
@@ -62,110 +53,13 @@ Item {
                 easing.type: Easing.OutQuad
             }
 
-
-            RowLayout {
-                id: row
-                width: parent.width - (defaultMargin * 2)
-                anchors.left: parent.left
-                anchors.right: parent.right
-                anchors.leftMargin: defaultMargin
-                anchors.rightMargin: defaultMargin
-                anchors.top: parent.top
-                anchors.topMargin: 20
-                spacing: 12
-
-                ColumnLayout {
-                    id: labelWrapper
-                    spacing: 4
-                    Layout.maximumWidth: parent.width - useSystemLanguageToggle.width - 16
-
-                    states: [
-                        State {
-                            when: useSystemLanguageToggle.checked
-                            PropertyChanges {
-                                target: labelDescription
-                                //% "Mozilla VPN will use your system’s default language."
```

Comment:
Is there any chance to just leave the strings and comments in the file, so we don't lose translations and are ready when this get merged back?

Reply:
Yes, done. Take a look again please.